### PR TITLE
enh: skip processing for empty response

### PR DESCRIPTION
**Pull request by kesselb**
## Summary

Skip the further processing if nothing to do. 

## TODO

- [x] CI

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by Altahrim — APPROVED** [review]

File: lib/private/Http/Client/DnsPinMiddleware.php [hunk 1]
```diff
@@ -128,6 +128,10 @@ public function addDnsPinning() {
 
 				$targetIps = $this->dnsResolve(idn_to_utf8($hostName), 0);
 
+				if (empty($targetIps)) {
+					throw new LocalServerException('No dns records found for ' . $hostName);
```
[the remark this answers is not in the document]
Comment (kesselb) [reply lib/private/Http/Client/DnsPinMiddleware.php]: Thanks, well spotted :+1:

**Review by ChristophWurst — APPROVED** [review]
Makes sense

**Comment by trumpandgod-droid** [comment]
Hacked by TRUMPANDGOD✨✨✨

**Comment by trumpandgod-droid** [comment]
HACKERONE GOT HACKED✨✨✨

**Pull request merged**